daphne/src/messages.rs

```diff
@@ -809,6 +817,218 @@ impl Decode for HpkeCiphertext {
     }
 }
 
```

Comment:
Can you add a comment noting that this message is defined in draft-wang-ppm-dap-taskprov-00? (Here and below.)

Reply:
fixed in next push